packages/Ludown/test/ludown.negative.test.suite.js

```diff
@@ -15,7 +15,7 @@ describe('Negative tests', function() {
     it('should show ERR message when no utterances are found for an intent', function(done) {
         exec(`node ${ludown} parse toluis --in ${TEST_ROOT}/testcases/bad3.lu`, (error, stdout, stderr) => {
             try {
-                assert.equal(stderr.includes('does not have list decoration. Prefix line with "-" or "+"'), true);
+                assert.equal(stderr.includes("syntax error: extraneous input 'i' expecting"), true);
```

Comment:
Not sure what the updated error message means. Can we make this more human readable? Also not sure why we are expecting 'i'

Reply:
The actual error message is "[ERROR] line 4:0 - line 4:1: syntax error: extraneous input 'i' expecting {<EOF>, MODEL_INFO, NEWLINE, QNA, HASH, DOLLAR, IMPORT_DESC}"

Reply:
Do you think the above error message ok for users? Or just change it to "**[ERROR] line 4:0 - line 4:1: syntax error: extraneous input 'i'**"?